tests/phpunit/tests/Checker/Checks/Plugin_Review_PHPCS_Check_Tests.php

```diff
@@ -22,7 +22,7 @@ public function test_run_with_errors() {
 
 		$this->assertNotEmpty( $errors );
 		$this->assertArrayHasKey( 'load.php', $errors );
-		$this->assertEquals( 2, $check_result->get_error_count() );
+		$this->assertEquals( 5, $check_result->get_error_count() );
```

Comment:
You added 1 line to the test file, why did the error count increase from 2 to 5? 🤔